### PR TITLE
A little varnish

### PR DESCRIPTION
**Pull request by cloveras**
A small xmas clean-up. Not 100 % perfect, but hopefully better.

### CODE REVIEW

File: docs/epayment.v1.yml [hunk 1]
```diff
@@ -290,19 +308,20 @@ components:
...
           example: '2017-07-21T17:32:28Z'
           pattern: '^((?:(\d{4}-\d{2}-\d{2})(T|t)(\d{2}:\d{2}:\d{2}(?:\.\d+)?))(Z|z|([+-](?:2[0-3]|[01][0-9]):[0-5][0-9])))$'
           nullable: true
           description: |-
-            Payment will expire at the given date and time. Format must adhere to RFC 3339. Value must be greater than 10 minutes in the future, and less than 24h in the future
-            If flow is QR, the QR-image will be returned as SVG+XML if qrFormat is not set.
+            Payment will expire at the given date and time. 
+            The format must adhere to RFC 3339.
+            The value must be greater than 10 minutes in the future, and less than 24h in the future
+            If flow is `QR`, the QR code image will be returned as `SVG+XML` if `qrFormat` is not set.
```
Comment (OFollan) [inline docs/epayment.v1.yml:321]: This line about QR doesn't belong here I assume 🤔

**Review by OFollan — APPROVED** [review]
LGTM 😄 
Other than the misplaced QR bit,

File: docs/epayment.v1.yml (hunk 1 above)
Comment (rebekaburnett) [reply docs/epayment.v1.yml:321]: @OFollan That line about QR was in the original yml file. Are you sure it doesn't belong?

**Review by rebekaburnett — APPROVED** [review]

File: docs/epayment.v1.yml (hunk 1 above)
Comment (OFollan) [reply docs/epayment.v1.yml:321]: Hehe - I guess that is a small error on main (probably from a sub-optimal merge). Line 321, the one with `If flow is `QR`, the QR code image will be returned as `SVG+XML` if `qrFormat` is not set.` 
It belongs somewhere in the docs, but not under the `expiresAt` description.

Comment (OFollan) [reply docs/epayment.v1.yml:321]: On line 333 we state that SVG+XML is the default value for QR Format. Perhaps this line can just be deleted entirely? Maybe @khenrichsen has a gut feeling about this?

File: docs/epayment.v1.yml [hunk 2]
```diff
@@ -33,12 +32,12 @@ paths:
...
     post:
       operationId: createPayment
-      summary: Create Payment
+      summary: Create payment
       tags:
         - CreatePayments
       description: Create a new payment
       requestBody:
-        description: New payment request body
+        description: New `PaymentRequest` body
```
Comment (tht13) [inline docs/epayment.v1.yml:40]: Didn't we change this to `CreatePaymentRequest`?

File: docs/epayment.v1.yml [hunk 3]
```diff
@@ -237,8 +249,8 @@ components:
         - NOK
       description: Currency code as defined in  ISO 4217. eg NOK for Norwegian kroner.
     CreatePaymentRequest:
-      title: Create Payment Request
-      description: Payment Request object
+      title: Create payment request
+      description: `PaymentRequest` object
```
Comment (tht13) [inline docs/epayment.v1.yml:253]:
```suggestion
      description: `CreatePaymentRequest` object
```

File: docs/epayment.v1.yml (hunk 2 above)
Comment (tht13) [inline docs/epayment.v1.yml:40]:
```suggestion
        description: New `CreatePaymentRequest` body
```

File: docs/epayment.v1.yml [hunk 4]
```diff
@@ -247,12 +259,18 @@ components:
...
+            Default is `false`.
         customer:
           $ref: '#/components/schemas/Customer'
         customerInteraction:
           type: string
-          description: The form of customer interaction to trigger the purchase
+          description: |-
+            The type of customer interaction that triggers the purchase.
+            `CUSTOMER_PRESENT` means that the customer is physically present at the
+            point of sale when the payment is made, tyøpically in a store.
```
Comment (tht13) [inline docs/epayment.v1.yml:273]:
```suggestion
            point of sale when the payment is made, typically in a store.
```

File: docs/epayment.v1.yml [hunk 5]
```diff
@@ -290,19 +308,20 @@ components:
           example: NATIVE_REDIRECT
           description: |-
             The flow for bringing to user to the Vipps Wallet payment confirmation screen.
-
-            If flow is `PUSH_MESSAGE` then one of `$.profile.sub` or `$.profile.phoneNumber` must be valid and is required.
+            If flow is `PUSH_MESSAGE` then one of `profile.sub` or `profile.phoneNumber` must be valid and is required.
```
Comment (tht13) [inline docs/epayment.v1.yml:311]:
```suggestion
            If flow is `PUSH_MESSAGE` then one of `customer.sub` or `customer.phoneNumber` must be valid and is required.
```

File: docs/epayment.v1.yml (hunk 1 above)
Comment (tht13) [reply docs/epayment.v1.yml:321]: I think this line actually belongs on line 311-312

File: docs/epayment.v1.yml [hunk 6]
```diff
@@ -324,16 +343,16 @@ components:
...
         - amount
         - paymentMethod
         - reference
         - returnUrl
         - userFlow
     CreatePaymentResponse:
-      title: Create Payment Response
-      description: Create Payment Response object
+      title: Create payment response
+      description: Create `PaymentResponse` object
```
Comment (tht13) [inline docs/epayment.v1.yml:355]:
```suggestion
      description: `CreatePaymentResponse` object
```

File: docs/epayment.v1.yml [hunk 7]
```diff
@@ -361,8 +380,8 @@ components:
       required:
         - reference
     GetPaymentResponse:
-      title: Get Payment Response
-      description: Get Payment Response object
+      title: Get payment response
+      description: Get `PaymentResponse` object
```
Comment (tht13) [inline docs/epayment.v1.yml:384]:
```suggestion
      description: `GetPaymentResponse` object
```

File: docs/epayment.v1.yml [hunk 8]
```diff
@@ -556,14 +574,17 @@ components:
       properties:
         scope:
           type: string
-          description: A space separated string list of requested user information in accordance with the OpenID Connect specification.
+          description: A space-separated string list of requested user information in accordance with the OpenID Connect specification.
```
Comment (tht13) [inline docs/epayment.v1.yml:577]: ironic that space-separated is not space separated ;)

File: docs/epayment.v1.yml [hunk 9]
```diff
@@ -199,23 +204,27 @@ components:
...
-          * The user subscription identifier received in the `profile.sub` property from the response body or from the Vipps Login API
+          * The user subscription identifier received in the `profile.sub` property from
+          the response body or from the Vipps Login API
           * Target users phone number in MSISDN format
       properties:
         phoneNumber:
           type: string
-          description: Mobile number in MSISDN format of the user who has to pay for the transation from Vipps. Currently vipps supports only Norwegian Mobile Numbers
+          description: |-
+            Mobile number in MSISDN format of the user who has to pay for the transation
```
Comment (rebekaburnett) [inline docs/epayment.v1.yml:214]:
```suggestion
            Mobile number in MSISDN format of the user who has to pay for the transaction
```

File: docs/epayment.v1.yml [hunk 10]
```diff
@@ -226,7 +235,10 @@ components:
           maximum: 65000000
           example: 1000
           format: int64
-          description: 'Integer value of price in the currencys monitary subunit (eg Norwegian Øre), or monitary unit where applicable (eg Japanese YEN). Type of monitary unit is defined in  ISO 4217.'
+          description: |-
+            Integer value of price in the currencys monitary subunit (eg Norwegian øre),
```
Comment (rebekaburnett) [inline docs/epayment.v1.yml:239]:
```suggestion
            Integer value of price in the currency's monetary subunit (e.g., Norwegian øre),
```

File: docs/epayment.v1.yml [hunk 11]
```diff
@@ -226,7 +235,10 @@ components:
           maximum: 65000000
           example: 1000
           format: int64
-          description: 'Integer value of price in the currencys monitary subunit (eg Norwegian Øre), or monitary unit where applicable (eg Japanese YEN). Type of monitary unit is defined in  ISO 4217.'
+          description: |-
+            Integer value of price in the currency's monetary subunit (e.g., Norwegian øre),
+            or monitary unit where applicable (eg Japanese YEN). The type of the monitary
```
Comment (rebekaburnett) [inline docs/epayment.v1.yml:240]:
```suggestion
            or monetary unit where applicable (e.g., Japanese YEN). The type of the monetary
```

File: docs/epayment.v1.yml [hunk 12]
```diff
@@ -706,8 +726,7 @@ components:
       type: string
       title: State
       description: |-
-        State of the Payment
-
+        State of the Payment.
         One of: 
         - CREATED : User has not yet acted upon the payment
         - ABORTED : User has aborted the payment before authorisation
```
Comment (rebekaburnett) [inline docs/epayment.v1.yml:732]:
```suggestion
        - ABORTED : User has aborted the payment before authorization
```

File: docs/epayment.v1.yml [hunk 13]
```diff
@@ -646,17 +674,17 @@ components:
             - CANCEL
             - CAPTURE
             - REFUND
-            - AUTHORISE
+            - AUTHORISE
```
Comment (rebekaburnett) [inline docs/epayment.v1.yml:677]: Are you going to eventually change that to AUTHORIZE?

File: docs/epayment.v1.yml [hunk 14]
```diff
@@ -646,17 +674,17 @@ components:
...
+          example: CREATE
           x-enum-varnames:
             - CREATE
             - ABORT
             - EXPIRE
             - CANCEL
             - CAPTURE
             - REFUND
-            - AUTHORISE
+            - AUTHORISE
```
Comment (rebekaburnett) [inline docs/epayment.v1.yml:687]: Are you going to eventually change that to AUTHORIZE?

File: docs/epayment.v1.yml (hunk 2 above)
Comment (rebekaburnett) [reply docs/epayment.v1.yml]: yes. I fix

File: docs/epayment.v1.yml (hunk 1 above)
Comment (rebekaburnett) [reply docs/epayment.v1.yml]: fixed

**Comment by rebekaburnett** [comment]
> A small xmas clean-up. Not 100 % perfect, but hopefully better.

A big xmas clean-up!

**Comment by rebekaburnett** [comment]
PS - I see a mixture of AUTHORISE and AUTHORIZE in the API.

**Pull request merged**